spec/rubocop/cop/security/io_read_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Security::IoRead, :config do
+  shared_examples 'offense' do |method|
+    it "registers an offense when using `#{method}`" do
+      expect_offense(<<~RUBY)
+        IO.#{method}
+        ^^ Use `File` instead of `IO`.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        File.#{method}
+      RUBY
+    end
+  end
+
+  shared_examples 'accepts' do |code|
+    it "does not register an offense when using `#{code}`" do
+      expect_no_offenses(<<~RUBY)
+        #{code}
+      RUBY
+    end
+
+    it "does not register an offense when using `#{code}`" do
+      expect_no_offenses(<<~RUBY)
+        #{code}
+      RUBY
+    end
```

Comment:
I think this is probably here twice by accident?

Reply:
Oops! This is a mistake and fixed it. Thanks for your review!